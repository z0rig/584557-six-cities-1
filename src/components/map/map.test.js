import React from "react";
import Enzyme, {mount} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Map from "./map.jsx";

const mock = [
  {
    id: `asdf0`,
    isPremium: true,
    imgSrc: `img/apartment-01.jpg`,
    price: 12000,
    period: `day`,
    inBookmarks: true,
    raiting: 5,
    title: `Old palace`,
    housingType: `Palace`,
    coord: [52.3909553943508, 4.85309666406198]
  }
];

Enzyme.configure({adapter: new Adapter()});

it(`Map correctly renders`, () => {
  const map = mount(<Map offers={mock} />);
  expect(map).toMatchSnapshot();
});
