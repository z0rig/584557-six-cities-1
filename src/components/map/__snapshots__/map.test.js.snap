// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map correctly renders 1`] = `ReactWrapper {}`;
